src/app/services/services.module.ts

```diff
@@ -45,11 +45,12 @@ import { PasswordRepromptService } from "../../services/passwordReprompt.service
 import { StateService } from "../../services/state.service";
 import { StateMigrationService } from "../../services/stateMigration.service";
 import { WebPlatformUtilsService } from "../../services/webPlatformUtils.service";
+import { PermissionsGuard as OrgPermissionsGuard } from "../organizations/guards/permissions.guard";
+import { NavigationPermissionsService as OrgPermissionsService } from "../organizations/services/navigation-permissions.service";
+import { RedirectToAdminGuard as OrgRedirectToAdminGuard } from "../organizations/services/redirect-to-admin.guard";
```

Comment:
Same question here, what do you think of just prefixing them with Organization and avoid the aliasing hassle.

Reply:
Same as above